http-client.ts

```diff
@@ -107,12 +116,7 @@ export class HttpClient implements Server.IHttpClient {
 				if (pipeTo) {
 					pipeTo.on("finish", () => {
 						this.$logger.trace("httpRequest: Piping done. code = %d", response.statusCode.toString());
-						if(!result.isResolved()) {
-							result.return({
-								response: response,
-								headers: response.headers
```

Comment:
You can shorten this call too and you might want to consider renaming `responseBody` to simply `body` in order to shorten it even further